### ISSUE TITLE
Can't run or pub build quickstart under 4.0.0-alpha

### ISSUE THREAD

**Issue by chalin** [dart-lang/angular#516]
Having upgraded to 4.0.0-alpha (via https://github.com/dart-lang/site-webdev/pull/809), we can no longer run or `pub build` even the [quickstart](https://webdev.dartlang.org/angular/guide/setup).

#### pub build

`pub build --web-compiler=dart2js --verbose` seems to just want to run forever, periodically printing

> FINE: [Fine from BuilderTransformer: GeneratorBuilder:[TemplateGenerator] on angular_test|lib/angular_test.dart]:
>   Not yet complete after XXXs, waiting on input(s) angular|lib/angular.dart, angular|lib/src/core/linker/view_ref.dart, angular|lib/src/debug/debug_app_view.dart, angular|lib/src/debug/debug_node.dart, angular|lib/di.dart, angular|lib/src/facade/exceptions.dart, angular|lib/src/core/application_ref.dart, angular|lib/src/core/change_detection/constants.dart, angular|lib/src/core/linker/app_view_utils.dart, angular|lib/src/mock/mock_location_strategy.dart, angular|lib/src/platform/location.dart

#### pub serve

`pub serve` works but the app fails to load and the following message is printed on the console:
```nocode
package:angular/src/core/reflection/reflection.dart:24 Exception: Cannot find reflection information on ApplicationRefImpl
```

cc @kwalrath

**Comment by alorenzen** [comment]
@chalin I don't think you should be renaming bootstrap(Component) to bootstrapStatic(Component). The transformer should be doing that for you. 

If you want to do it manually, then you'd also have to import main.template.dart and pass initReflector to bootstrapStatic.

**Comment by chalin** [comment]
Ah, ok. That has allowed me to `pub serve` and then successfully load/run the quickstart app. But `pub build` still seems to just take forever &mdash; I've let it run for at least 3 min in an attempt to build quickstart.

**Comment by chalin** [comment]
@matanlurey - what info is needed? The doc examples updated to 4.0.0-alpha can be found under the `4-dev` branch; e.g. https://github.com/dart-lang/site-webdev/tree/4-dev/examples/ng/doc. I'm still getting the `pub build` error mentioned in the opening commment.

**Comment by matanlurey** [comment]
@chalin We just released 4.0.0-alpha+1 if you can give that a try.

**Comment by chalin** [comment]
Yes, I can successfully build and serve the quickstart app now (which was the original issue being reported here). Great!

E2E tests are still failing though. I'll investigate tomorrow (and open a separate issue as necessary).
